Assets/Scripts/2D/ModalPanels/ModalPanelScript.cs

```diff
@@ -12,7 +12,7 @@ public virtual void SetVisible(bool state)
         if (!state && !gameObject.activeInHierarchy)
             return; // There's no need to make this dialog invisible if it already is. Also, we want to avoid disabling the canvas group if it's being enabled by another dialog
 
-        ModalPanelCanvasGroup.gameObject.SetActive(state);
+        ModalPanelCanvasGroup.GetComponent<ModalActivationScript>().Activate(state);
```

Comment:
What is the difference? I don't know and was just wondering.

Reply:
The idea is that calling the Activate() function for ModalActivationScript will do more than just enable/enable the gameObject. In this case, it will also do 'Manager.DisableShortcuts = state;'. That way, every time a modal dialog is enabled it will also prevent keyboard shortcuts from being activated.